internal/policy/verify_test.go

```diff
@@ -2216,6 +2216,64 @@ func TestVerifyEntry(t *testing.T) {
 		assert.Nil(t, err)
 	})
 
+	t.Run("unsuccessful verification with higher threshold but using GitHub approval due to invalid app key", func(t *testing.T) {
+		t.Setenv(dev.DevModeKey, "1")
+		t.Setenv(tufv02.AllowV02MetadataKey, "1")
+
+		repo, state := createTestRepository(t, createTestStateWithThresholdPolicyAndGitHubAppTrust)
+
+		currentAttestations, err := attestations.LoadCurrentAttestations(repo)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		// This is using the jane.doe signer
+		commitIDs := common.AddNTestCommitsToSpecifiedRef(t, repo, refName, 1, gpgKeyBytes)
+
+		commitTreeID, err := repo.GetCommitTreeID(commitIDs[0])
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		// Create authorization for this change using john.doe trusted as approver
+		githubAppApproval, err := attestations.NewGitHubPullRequestApprovalAttestation(refName, gitinterface.ZeroHash.String(), commitTreeID.String(), []string{"john.doe"}, nil)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		// This signer for the GitHub app is NOT trusted in the root setup by
```

Comment:
this is a copy of a previous successful test that changes the key used to sign the app attestation to one that's not trusted